test/test_tensordict.py

```diff
@@ -764,6 +764,81 @@ def test_set_nontensor(self, td_name):
         td.set("numpy", r.numpy())
         torch.testing.assert_allclose(td.get("numpy"), r)
 
+    def test_convert_ellipsis_to_idx(self, td_name):
+        torch.manual_seed(1)
+        batch_size = [3, 4, 5, 6, 7]
+
+        ellipsis_idx = [..., (0, ..., 0), (..., 0), (0, ...), (slice(1, 2), ...)]
+
+        error_idx = [(..., 0, ...), (0, ..., 0, ...)]
+
+        expected_idx = [
+            (slice(None), slice(None), slice(None), slice(None), slice(None)),
+            (0, slice(None), slice(None), slice(None), 0),
+            (slice(None), slice(None), slice(None), slice(None), 0),
+            (0, slice(None), slice(None), slice(None), slice(None)),
+            (slice(1, 2), slice(None), slice(None), slice(None), slice(None)),
+        ]
+
+        for i in range(len(ellipsis_idx)):
+            assert (
+                convert_ellipsis_to_idx(ellipsis_idx[i], batch_size) == expected_idx[i]
+            )
+
+        for i in range(len(error_idx)):
+            with pytest.raises(RuntimeError):
+                _ = convert_ellipsis_to_idx(error_idx[i], batch_size)
+
+    def test_getitem_ellipsis(self, td_name):
+        torch.manual_seed(1)
+        td = TensorDict(
+            {"a": torch.randn(1, 3, 4, 5, 6), "b": torch.randn(1, 3, 4, 5)},
+            batch_size=[1, 3, 4],
+        )
```

Comment:
The purpose of td_name is to check that the indexing works for every tensordict class. Here we only test TensorDict